webapp/src/components/MeetingQuickView.tsx

```diff
@@ -0,0 +1,182 @@
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Alert,
+  Link as MuiLink,
+  Stack,
+  TextField,
+  Typography,
+} from '@mui/material';
+import CheckCircleOutlineIcon from '@mui/icons-material/CheckCircleOutline';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import { Link as ReactRouterLink } from 'react-router-dom';
+import { LoadingButton } from '@mui/lab';
+import React, { FormEventHandler, useContext, useState } from 'react';
+
+import { EntityId, Meeting as APIMeeting } from '../types/api';
+import { formatDate, formatTime } from '../helpers/dateTime';
+import SessionContext from './SessionContext';
+import useApiData from '../helpers/useApiData';
+
+interface MeetingQuickViewProps {
+  meeting: APIMeeting;
+}
+
+const MeetingQuickView = ({ meeting }: MeetingQuickViewProps) => {
+  const { principalId } = useContext(SessionContext);
+  const [changedMeetingNoteIds, setChangedMeetingNoteIds] = useState<
+    EntityId[]
+  >([]);
+  const [isExpanded, setIsExpanded] = useState(false);
+  const [isSavingMeetingNote, setIsSavingMeetingNote] = useState(false);
+  const [isSuccessIndicated, setIsSuccessIndicated] = useState(false);
+  const [meetingNoteText, setMeetingNoteText] = useState('');
+  const [saveMeetingNoteError, setSaveMeetingNoteError] = useState(null);
+
+  const { data: meetingWithNotes, error } = useApiData<APIMeeting>({
+    deps: [changedMeetingNoteIds],
+    path: `/meetings/${meeting.id}`,
+    sendCredentials: true,
+  });
+  if (error) {
+    return <div>There was an error loading the meeting.</div>;
+  }
+  if (!meetingWithNotes) {
+    return null;
```

Comment:
Should the summary still be shown even while it's loading the notes? The summary only needs data from the props in order to be rendered.

Reply:
Yes, this line was removed as part of some changes made from one of your conversations about using the `deps` and `shouldFetch` arguments of the `useAPIData` hook.